spring-boot/admin/src/main/java/eu/coatrack/admin/WebConfig.java

```diff
@@ -59,13 +60,8 @@ public void addInterceptors(InterceptorRegistry registry) {
     @Override
     public void configureContentNegotiation(ContentNegotiationConfigurer configurer) {
         // Simple strategy: only path extension is taken into account
-        configurer.favorPathExtension(true).
-                favorParameter(false).
-                parameterName("mediaType").
-                ignoreAcceptHeader(false).
-                useJaf(false).
-                defaultContentType(MediaType.APPLICATION_JSON).
-                mediaType("xml", MediaType.APPLICATION_XML).
-                mediaType("json", MediaType.APPLICATION_JSON);
+        configurer.favorPathExtension(true).favorParameter(false).parameterName("mediaType").ignoreAcceptHeader(false)
```

Comment:
This re-formatting makes the code harder to read, as the newlines were added on purpose. Is it possible to configure the formatter to "not join lines".

Reply:
take into account the line is "join" in the unify form, you can always clone and format in your most preferable IDE and formating, do whatever you want and commit which will reformat joining the lines

Reply:
In general I agree: everyone can use their own IDE formatting and then the maven plugin will unify the formatting. That approach sounds fine.

The only problem I see: sometimes it makes sense to manually add newlines to make the code easier to read, which also helps reviewers during pull request reviews. But these newlines get lost during the auto-formatting. Therefore I would suggest a slightly changed config, which contains the setting: `<setting id="org.eclipse.jdt.core.formatter.join_wrapped_lines" value="false"/>` (by default this is `true`).

See my pull request: #143